### PR TITLE
feat: quickly acces candidate

### PR DESCRIPTION
**Pull request by laampui**
quickly select candidate by pressing M-0, M-1... in acm-mode-keymap
录屏_emacs_20220730134622

reference:
 https://github.com/company-mode/company-mode/blob/73a3bdea0a82731ba8fb5890b6cdcba5b9dd44b7/company.el#L705-L813

### CODE REVIEW

File: acm/acm.el [hunk 1]
```diff
@@ -671,6 +677,7 @@ influence of C1 on the result."
         ;; Build candidate line.
         (setq candidate-line
               (concat
+               (when acm-enable-quick-access (nth item-index acm-quick-access-keys))
                icon-text
```
Comment (manateelazycat) [inline acm/acm.el:681]: Show index right of icon will more neat, can you swap order?

**Review by manateelazycat — CHANGES_REQUESTED** [review]

**Comment by manateelazycat** [comment]
@laampui Thanks, awesome works, can you swap order of icon and index? Thanks.

**Comment by laampui** [comment]
image

**Comment by manateelazycat** [comment]
thanks，wonderful patch

**Pull request merged**